parsely-referrer-blacklist.php

```diff
@@ -0,0 +1,26 @@
+<?php
+$blacklist = array(
+  "softpopads",
+  "74.55.82.98",
+  "myonlinestats",
+  "fhmnet",
+  "longurl",
+  "74.55.82.102",
+  "carbon",
+  "cpc.rapidhits",
+  "4webmasters.org",
+  "bqao",
+  "adoncorst",
+  "kirilogne",
+  "reibeneal",
+  "trafficmonetize",
+  "googlesyndication",
+  "74.55.82.101",
+  "webmonetizer",
+  "traffic2money",
+  "boost-my-site",
+  "buyfastwebtraffic",
+  "qobk"
+);
+
+$blacklist = array_map('strtolower', $blacklist);
```

Comment:
this `strtolower` call is redundant given they are all stored in lowercase.....?